mobile/android/build.gradle

```diff
@@ -9,6 +9,7 @@ buildscript {
         classpath 'com.android.tools.build:gradle:7.2.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath 'com.google.gms:google-services:4.3.15'
+        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.8.1'
```

Comment:
@katendejericho5 was this added by `flutterfire`??

Reply:
Yes , it was added automatically

Reply:
Great. Thanks @katendejericho5